src/onInit/calculatePopulationSize.js

```diff
@@ -0,0 +1,5 @@
+import { set } from 'd3';
```

Comment:
Spencer, is everything in your life modular?

Reply:
just my home